<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-I3EPC00" failures="1" tests="6" name="com.reqers.testcases.GetUserListTest" time="0.049" errors="0" timestamp="2024-05-13T12:40:39 IST" skipped="0">
  <testcase classname="com.reqers.testcases.GetUserListTest" name="checkResponseBody" time="0.004">
    <system-out>
      <![CDATA[Response Body is => io.restassured.internal.RestAssuredResponseImpl@4c6a4ffd]]>
    </system-out>
  </testcase> <!-- checkResponseBody -->
  <testcase classname="com.reqers.testcases.GetUserListTest" name="checkStatusLine" time="0.001">
    <system-out>
      <![CDATA[Status Line is => HTTP/1.1 200 OK]]>
    </system-out>
  </testcase> <!-- checkStatusLine -->
  <testcase classname="com.reqers.testcases.GetUserListTest" name="checkServerType" time="0.003">
    <system-out>
      <![CDATA[Server Type is => nginx/1.21.6]]>
    </system-out>
  </testcase> <!-- checkServerType -->
  <testcase classname="com.reqers.testcases.GetUserListTest" name="checkContentType" time="0.002">
    <system-out>
      <![CDATA[Content Type is => application/json]]>
    </system-out>
  </testcase> <!-- checkContentType -->
  <testcase classname="com.reqers.testcases.GetUserListTest" name="checkStatusCode" time="0.001">
    <system-out>
      <![CDATA[Status Code is => 200]]>
    </system-out>
  </testcase> <!-- checkStatusCode -->
  <testcase classname="com.reqers.testcases.GetUserListTest" name="checkResponseTime" time="0.038">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.reqers.testcases.GetUserListTest.checkResponseTime(GetUserListTest.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
    <system-out>
      <![CDATA[Response Time is => 1468]]>
    </system-out>
  </testcase> <!-- checkResponseTime -->
  <system-out>
    <![CDATA[Response Body is => io.restassured.internal.RestAssuredResponseImpl@63f855b
Status Code is => 404
Status Line is => HTTP/1.1 404 Not Found
Response Body is => io.restassured.internal.RestAssuredResponseImpl@7fad214a
Status Code is => 200
Status Line is => HTTP/1.1 200 OK
Response Body is => io.restassured.internal.RestAssuredResponseImpl@3443e476
Status Code is => 200
Status Line is => HTTP/1.1 200 OK
Content Type is => application/json; charset=utf-8
Response Body is => io.restassured.internal.RestAssuredResponseImpl@3e377967
Response Time is => 137
Response Time is < 2000 
Server Type is => cloudflare
Status Code is => 200
Status Line is => HTTP/1.1 200 OK
Response Body is => io.restassured.internal.RestAssuredResponseImpl@9accff0
Status Code is => 200
Status Line is => HTTP/1.1 200 OK
Content Type is => application/json
Response Body is => io.restassured.internal.RestAssuredResponseImpl@4c6a4ffd
Response Time is => 1468
Server Type is => nginx/1.21.6
Status Code is => 200
Status Line is => HTTP/1.1 200 OK
Content Type is => application/json; charset=utf-8
Response Body is => io.restassured.internal.RestAssuredResponseImpl@5042e3d0
Response Time is => 164
Response Time is < 2000 
Server Type is => cloudflare
Status Code is => 200
Status Line is => HTTP/1.1 200 OK
Response Body is => io.restassured.internal.RestAssuredResponseImpl@62a54948
Status Code is => 201
Status Line is => HTTP/1.1 201 Created
Content Type is => application/json; charset=utf-8
Response Body is => io.restassured.internal.RestAssuredResponseImpl@251c4280
Response Time is => 449
Response Time is < 2000 
Server Type is => cloudflare
Status Code is => 404
Status Line is => HTTP/1.1 404 Not Found
Response Body is => io.restassured.internal.RestAssuredResponseImpl@4877919f
Status Code is => 200
Status Line is => HTTP/1.1 200 OK
Response Body is => io.restassured.internal.RestAssuredResponseImpl@49c099b
Status Code is => 400
Status Line is => HTTP/1.1 400 Bad Request]]>
  </system-out>
</testsuite> <!-- com.reqers.testcases.GetUserListTest -->
